packages/lit-dev-content/site/tutorials/content/intro-to-lit/01.md

```diff
@@ -19,7 +19,7 @@ In Lit, most things start with defining a _component_. Here we've given you a st
 
     {% endswitchable-sample %}
 
-    The `MyElement` class provides the implementation for your new component, and the `@customElement` decorator registers it with the browser as a new element type named `my-element`.
+    The `MyElement` class provides the implementation for your new component. <ts-js><span slot="ts">The <code>@customElement</code> decorator registers it with the browser as a new element type named <code>my-element</code>.</span><span slot="js">The <code>customElements.define</code> call registers it with the browser as a new element type named <code>my-element</code>.</span></ts-js>
```

Comment:
The HTML minifier unfortunately deletes this space in the prod build (it doesn't run in the dev build). We just need to add a `&nbsp;` to force a space there, minification or not.

```suggestion
    The `MyElement` class provides the implementation for your new component.&nbsp;<ts-js><span slot="ts">The <code>@customElement</code> decorator registers it with the browser as a new element type named <code>my-element</code>.</span><span slot="js">The <code>customElements.define</code> call registers it with the browser as a new element type named <code>my-element</code>.</span></ts-js>
```